src/Orchard.Cms.Web/Modules/Orchard.Contents/Controllers/AdminController.cs

```diff
@@ -447,7 +447,24 @@ public Task<IActionResult> EditPOST(int id, string returnUrl)
 
         private async Task<IActionResult> EditPOST(int id, string returnUrl, Func<ContentItem, Task> conditionallyPublish)
         {
-            var contentItem = await _contentManager.GetAsync(id, VersionOptions.DraftRequired);
+            var contentItem = await _contentManager.GetAsync(id, VersionOptions.Latest);
+
+            if (contentItem == null)
+            {
+                return NotFound();
+            }
+
+            ContentTypeDefinition typeDefinition = _contentDefinitionManager.GetTypeDefinition(contentItem.ContentType);
+            var draftRequired = (!contentItem.Published) || (typeDefinition.Settings.ToObject<ContentTypeSettings>().Draftable);
```

Comment:
As i remember it is because if Draftable it acts as before.
OR if not Published, anyway it needs a new version (e.g the 1st and only one), so do as before.
I've done some 1st tests with Draftable = true / false, seems ok.